File: src/main/java/org/adempiere/exceptions/BPartnerNoShipToAddressException.java
package org.adempiere.exceptions;

import org.compiere.model.I_C_BPartner;

/**
 * Thrown when Ship To Address is required for a BPartner but not found.
 * @author Teo Sarca, www.arhipac.ro
 */
public class BPartnerNoShipToAddressException extends BPartnerException
{
	/**
	 * 
	 */
	private static final long serialVersionUID = 4388496060894704270L;
	public static final String AD_Message = "BPartnerNoShipToAddress";
	
	public BPartnerNoShipToAddressException(I_C_BPartner bp)
	{
		super(AD_Message, bp);
	}
}

File: src/main/kotlin/org/adempiere/exceptions/BPartnerNoShipToAddressException.kt
package org.adempiere.exceptions

import org.compiere.model.I_C_BPartner

/**
 * Thrown when Ship To Address is required for a BPartner but not found.
 * @author Teo Sarca, www.arhipac.ro
 */
class BPartnerNoShipToAddressException(bp: I_C_BPartner) : BPartnerException(AD_Message, bp) {
    companion object {
        /**
         *
         */
        private val serialVersionUID = 4388496060894704270L
        val AD_Message = "BPartnerNoShipToAddress"
    }
}